cli/cli.go

```diff
@@ -18,8 +18,7 @@ const (
 )
 
 const (
-	MAIN_HELP_TEMPLATE = `
-Usage {{.Name}} [GLOBAL_OPTIONS] command [COMMAND_OPTIONS] [PARAMS]
+	MAIN_HELP_TEMPLATE = `Usage {{.Name}} [GLOBAL_OPTIONS] command [COMMAND_OPTIONS] [PARAMS]
```

Comment:
Does this mean the empty line before "Usage dp2 ..." is removed? Could this be put in a separate commit?

Reply:
Oups might have wrongly added this to the PR

(Got mixed up with the vs code go extension doing auto format with goeverywhere)

I might clean the PR branch to discard the reformat stuff that got in the way.